src/openrct2-ui/windows/ViewClipping.cpp

```diff
@@ -343,6 +344,16 @@ class ViewClippingWindow final : public Window
 
     void OnOpen() override
     {
+        // only occurs for the first time when a scenario starts and when this window opens.
+        // resets the clipping values to the default values
+        if (firstStart)
+        {
+            firstStart = false;
+            int zero = 0;
+            gClipSelectionA = { zero, zero };
+            gClipSelectionB = { MAXIMUM_TILE_START_XY, MAXIMUM_TILE_START_XY };
+        }
+
```

Comment:
There is an issue that occurs when the cut-away view is used on the current scenario, then a new scenario is loaded.
It previously did not reset the values. This means if the cut-away view is opened up in the new scenario, it loads the old values that were used in the previous scenario. This can be problematic if the new scenario uses a smaller map than the previous scenario.

Reply:
gClipSelectionA and gClipSelectionB are global variables that are defined in Paint.cpp and Paint.h respectively and seem to be used to store the boundaries for the cut-away view. I would rather handle this current pull request first. I can either open up an issue to highlight the need to refactor these global variables or open another pull request after this.

Could be a good first issue for someone else,